src/app/@ansyn/ansyn/modules/status-bar/components/advanced-search/advanced-search.component.ts

```diff
@@ -97,8 +99,20 @@ import { SearchAction } from '../../actions/status-bar.actions';
 	}
 
 	search(): void {
-		this.store.dispatch(new SearchAction(this.getCurrentAdvancedSearchParameters()));
-		this._parent.close();
+		const params = this.getCurrentAdvancedSearchParameters();
+		if (this.isValid(params)) {
+			this.store.dispatch(new SearchAction(params));
+			this._parent.close();
+		} else {
+			this.showMessage = true;
+		}
+	}
+
+	isValid(params: IAdvancedSearchParameter): boolean {
+		return params?.registeration?.length > 0
```

Comment:
I thinks it's ok that the registration be empty.

Reply:
This is not what I'm checking here.  Validation will fail only if **all** search fields are empty.